app/javascript/components/service-dialog-from-form/service-dialog-from.schema.js

```diff
@@ -18,7 +18,7 @@ const asyncValidatorDebounced = debouncePromise(asyncValidator);
 const serviceDialogFromOtSchema = ({
   fields: [{
     component: componentTypes.TEXT_FIELD,
-    name: 'label',
+    name: 'name',
```

Comment:
Api expect label not name this won't work.

Reply:
@Hyperkid123 i verified creation of service dialog in UI, not sure how that worked then.

Reply:
Did you run `bin/webpack` to compile the JS?

Reply:
@Hyperkid123 you are right after running webpack it didn't work.

Reply:
@Hyperkid123 @epwinchell looking at Dialogs db table, there is no name field in the table, so would it make sense to change the label of the Field to "Service Dialog Label" instead of renaming html element.